packages/igx-templates/Update.ts

```diff
@@ -44,13 +44,14 @@ export async function updateWorkspace(rootPath: string): Promise<boolean> {
 	const fs: IFileSystem = App.container.get(FS_TOKEN);
 	for (const packageDef of UPGRADEABLE_PACKAGES) {
 		if (resolvePackage(packageDef.licensed, packageDef.trial) === packageDef.licensed) {
+			Util.log("Your app is already using packages from the Infragistics registry. You are good to go.", "green");
```

Comment:
Okay that's in the  for loop though, so if the project has two already updates packages.. I get two messages?
I realize there's not a decent place to plug this in, will have to update the refactor branch itself first.